src/contracts/core/AVSDirectory.sol

```diff
@@ -827,6 +827,42 @@ contract AVSDirectory is
         return allocatableMagnitude;
     }
 
+    /*
+    * @param operator the operator to get the slashable magnitude for
+    * @param strategy the strategy to get the slashable magnitude for
+    * @param operatorSet the operatorSet to get the slashable magnitude for
+    * 
+    * @return the current slashable magnitude of the given operator, strategy, and operatorSet
+    */
+    function getCurrentSlashableMagnitude(
+        address operator,
+        IStrategy[] calldata strategy
+    ) public view returns (OperatorSet[] memory, IStrategy[] memory, uint64[] memory) {
+        OperatorSet[] memory operatorSets = operatorSetsMemberOf(operator, 0, inTotalOperatorSets(operator));
+        uint256 maxValues = operatorSets.length * strategy.length;
+        OperatorSet[] memory returnOperatorSets = new OperatorSet[](maxValues);
+        IStrategy[] memory returnStrategies = new IStrategy[](maxValues);
+        uint64[] memory slashableMagnitude = new uint64[](maxValues);
```

Comment:
make this a 2d array of `[strategies.length][operatorSets.length]`

Reply:
0ea5394635402ed033a2adb513dbb40d38aab798